### PR TITLE
Notifications improvements

### PR DESCRIPTION
**Pull request by samiuelson**
* Adds support for additional values send through firebase cloud message: channel_name, message_text
* Minor code clean-up

Should be merged after LLC 1.16.2 is released

### CODE REVIEW

File: livedata/build.gradle [hunk 1]
```diff
@@ -139,7 +139,7 @@ afterEvaluate {
                 // Adds sources as separate jars.
                 artifact androidSourcesJar
                 // Custom attributes of the publication
-                groupId = project.group
+                groupId = 'com.github.getstream'
```
Comment (JcMinarro) [inline livedata/build.gradle:142]: Why are we changing it?

Comment (samiuelson) [reply livedata/build.gradle:142]: If `publishToMavenLocal` is done with 'com.github.GetStream' group there is a metadata error while fetching the artifact in the target project. Or is it only locally on my end?

Comment (JcMinarro) [reply livedata/build.gradle:142]: I didn't have this problem before.
BTW, Can't you change the `project.group` value?

Comment (samiuelson) [reply livedata/build.gradle:142]: Not sure how jitpack will react. I will probably get stuck finding repo on github

**Review by JcMinarro — APPROVED** [review]

**Pull request merged**